README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
> i do not understand what you are suggesting rather lots of changes in all the repositories. means.

all in all, to enforce your suggestion, we need to re-write the rules. all of `README.md` must be updated. for what? for the completeness? as I wrote, I see no points.